src/entity/units/ancient.ts

```diff
@@ -16,7 +16,7 @@ const unit: UnitSource = {
     subtype: UnitSubType.RANGED_RADIUS,
   },
   unitProps: {
-    damage: 3,
+    damage: 10,
```

Comment:
My intent is that the ancient is a small, persistent danger.  If you don't deal with them they'll become a problem but aren't an immediate threat.  With the higher base health, I can understand this mostly.  Can you provide more context as to why this change?

Reply:
It's primarily due to increased max health. They also cast a bit less due to mana changes